hnn_core/network.py

```diff
@@ -965,6 +965,8 @@ def _instantiate_drives(self, tstop, n_trials=1):
             for drive in self.external_drives.values():
                 event_times = list()  # new list for each trial and drive
                 for drive_cell_gid in self.gid_ranges[drive['name']]:
+                    drive_cell_gid_offset = (drive_cell_gid -
```

Comment:
Does this mean that all drives default to a seed of `0` when `event_seed` isn't defined?

I just want to make sure we aren't running into another form of #414, where different drives have the same spike times if it's not explicitly set.

Reply:
Thinking further, the test for the simulation comparison uses `add_drives_from_params=True`. Two tests come to mind:

1) simulation comparison `add_drives_from_params=False` (perhaps a future PR)
2) Confirmation that adding drives in different orders instantiates the same drive objects, since that was the problem in #454

Reply:
well, `event_seed` is by default 2. This PR just makes sure that giving `event_seed=2` always produces the same result. Now, we could change the default `event_seed` to be `None` and this would be compatible with other software like MNE and sklearn. The behavior when it is `None` would not be reproducible though ...

Reply:
I agree about adding the tests, was just being lazy ;-) Thanks for catching that!

Reply:
also using `None` as default would highlight the fact that multiple trials are needed to make a claim.

Reply:
I agree that emphasizing multiple trials should be more visible. I think we're guilty of not pushing it because it's not useful for debugging purposes.

The more I think about it using `None` as a default does sound appealing. I've definitely been guilty of reading too much into the randomness of a single simulation, and see the result disappear with new seeds.

Reply:
+1 for this! We might want to consider adding a big fat warning sign on our API doc page that highlights this if we implement it in the next release though.